src/Quartz/Configuration/TriggerExtensions.cs

```diff
@@ -2,6 +2,13 @@ namespace Quartz;
 
 public static class TriggerExtensions
 {
+    /// <summary>
+    /// Sets up a daily schedule with a default interval of *every-minute* unless overridden by the action
+    /// by calling WithInterval / WithIntervalInHours / WithIntervalInMinutes.
+    ///
+    /// For the sake of clarity, you probably want to use WithDailyConfiguredTimeIntervalSchedule instead.
+    /// </summary>
+    [Obsolete("Use WithDailyConfiguredTimeIntervalSchedule with default interval params instead.")]
```

Comment:
Not sure if obsoleting this is ideal?.  
But adding the doc comments does benefit in describing what is occurring 👍🏻

Reply:
My worry is that people won't read the docs comment. The behaviour I ran into (and the folks on StackOverflow also got burnt by) _was_ documented online in some places, but I didn't stumble across it until it was too late. If you obsolete the whole method, you a) don't break backwards-compat _completely_ but more importantly b) warn everyone with a half-decent IDE that what they're doing may be dangerous and prompt them to read the method docs.

My suggestion would be to _either_ change the default interval (to e.g. every 24h) and forego the `[Obsolete]` addition, or keep it because migrating folks over to `WithDailyConfiguredTimeIntervalSchedule` where the interval _has_ to be specified up-front is safer

Reply:
I think my concern mainly came about backwards compat, but as you state, obsolete won't break backwards compat.  And perhaps in v4/5 it can be removed.  
Given the confusion you/others have found, makes sense to be more explicit 👍🏻

Reply:
Sorry, I know it's a bit messy!